src/pages/NotFound.tsx

```diff
@@ -0,0 +1,57 @@
+import { styled } from "@mui/material";
+import emptyPlate from "../assets/images/plate.svg";
+import { useTranslation } from "react-i18next";
+
+const StyledImage = styled("img")(({ theme }) => ({
+  height: "20rem",
+  filter:
+    "invert(94%) sepia(0%) saturate(0%) hue-rotate(148deg) brightness(93%) contrast(90%)",
```

Comment:
why these exact values? 
is this image used elsewhere?

Reply:
I've read that I can only change the color of an SVG image by applying a filter to the `<img>`  ( or directly in the SVG ) so, I've used an online converter to translate the hex color value into filter values to achieve the get color I want. (option with `color: value` doesnt seem to work)

This image isn't used elsewhere yet, but I'm considering using it to display information on other pages when there is no content. >,<

Reply:
you might want to look into this 
https://dev.to/receter/so-easy-svg-with-currentcolor-in-create-react-app-1aid

Reply:
Thank you, works like a charm! But could You please review the changes? I want to be sure that they are correct 😄